test/chart/setDomain.js

```diff
@@ -0,0 +1,61 @@
+import jsdom from 'jsdom';
```

Comment:
tests look good!

Reply:
I guess we could take it a step further and test all the various use cases in the huge if/then/else from setDomain(). Maybe make an issue for later?

Reply:
Yeah, the domain algorithm probably warrants some more comprehensive testing: https://github.com/RhoInc/Webcharts/issues/196